imapserver/status.go

```diff
@@ -45,6 +45,14 @@ func (c *Conn) handleStatus(dec *imapwire.Decoder) error {
 		return err
 	}
 
+	// Check if AppendLimit was requested and the session supports it
+	if options.AppendLimit && data.AppendLimit == nil {
+		if appendLimitSession, ok := c.session.(SessionAppendLimit); ok {
+			limit := appendLimitSession.AppendLimit()
+			data.AppendLimit = &limit
+		}
+	}
```

Comment:
This prevents backends from returning a per-mailbox append limit.

Backends should be able to populate `data.AppendLimit` on their own already, so I don't think we need this?

Reply:
per-mailbox limit is a very edge functionality. While it is possible per RFC, I would not follow it blindly. Dovecot has this as global for instance too. 

Nevertheless, agreed, lets move that to implementing server

Reply:
(updated)